OrbitGl/Batcher.cpp

```diff
@@ -94,18 +94,18 @@ void Batcher::AddTriangle(Vec3 v0, Vec3 v1, Vec3 v2, Color color,
 }
 
 PickingUserData* Batcher::GetUserData(PickingID a_ID) {
```

Comment:
second commit: please reformat commit message to have title limited to 50 characters \n\n and then description.